core/src/main/scala/cats/syntax/all.scala

```diff
@@ -58,5 +58,6 @@ trait AllSyntaxBinCompat0
     extends UnorderedTraverseSyntax
     with ApplicativeErrorExtension
     with TrySyntax
+    with BinestedSyntax
```

Comment:
hmmm, this might break bin compat with 1.1.0. 
Would you mind also change the bin compat version [here](https://github.com/typelevel/cats/blob/master/build.sbt#L207) to `1.1.0`? That should capture this error.

Reply:
Ah cool. I'll rebase on that.

Reply:
I see that sticking it on `AllSyntaxBinCompat1` is fine. Will change to that...